config/index.js

```diff
@@ -8,6 +8,20 @@ const config = {
   MONGO_URI: process.env.MONGO_URI || 'mongodb://localhost:27017/mydatabase',
   session_secret: process.env.SESSION_SECRET || crypto.randomBytes(64).toString('hex'),
   tokenTimeOut: 1000,
+
+  // email service config
+  mail: {
+    host: process.env.MAIL_HOST || 'smtp.qiye.aliyun.com',
```

Comment:
Bad idea to hard code default configuration in your code, sometimes without attention you may expose some important information such as db password, etc. to external user and cause potential security threat.

Reply:
So in here what should I to improve the security?

Reply:
> So in here what should I to improve the security?

just an idea, might need further discuss: save the config in another file without uploading to github, and access the config file in your code here.

Reply:
Ying propose a better solution that saves them in the configuration file. For instance, the .env.